### DISCUSSION TITLE
Feedback on Profile link and achievements

### DISCUSSION THREAD

**Discussion by urielgarciarivas** [community/community#19324] [Other Feature Feedback, Questions, & Ideas]
Hello! Its my first time posting here, I just want to share two things:

- It took me off guard when I clicked on the **Send feedback** link, right below my achievements, because I was expecting a simple `textarea` and a `submit` button. You could consider redirecting right to the **Start a new discussion** page.

- I don't know if it was intended, but the **Pull Shark achievement** also worked with my own repositories (e.g., I created a repository, then I created a pull request on it, then I myself merged that pull request). If it was actually intended, I suggest you to do a separate achievement for external repositories.

That's all 😁

**Comment by rodukov** [comment]
**I would not make a button to watch discussions**, and I think that the github is right. Because there are such discussions not only in the organization of the github community
And about the second, yes, I myself don’t understand why the github did it. I think the **pull shark achievement should only be given by other repositories that we don't own.** This is how the meaning of all these achievements is preserved.
_If it's not hard, you can make the question the main one. I will be pleased_
